<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>633</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>FairWind</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>342</width>
      <height>238</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QStackedWidget { background-image: url(:/resources/images/carbon_base.png); }</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page_Apps">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_Apps">
      <item>
       <widget class="QScrollArea" name="scrollArea_Apps">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents_Apps">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>8</y>
           <width>316</width>
           <height>196</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="page_Settings">
     <layout class="QVBoxLayout" name="verticalLayout_Setting">
      <item>
       <widget class="QTabWidget" name="tabWidget_Settings">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_Settings">
         <attribute name="title">
          <string>Settings</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_Settings">
          <item>
           <widget class="QScrollArea" name="scrollArea_Settings">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents_Settings">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>179</width>
               <height>157</height>
              </rect>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_Settings_Buttons">
            <item alignment="Qt::AlignTop">
             <widget class="QPushButton" name="pushButton_Settings_Cancel">
              <property name="text">
               <string>Cancel</string>
              </property>
              <property name="default">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignTop">
             <widget class="QPushButton" name="pushButton_Settings_Save">
              <property name="text">
               <string>Save</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_Data_Connections">
         <attribute name="title">
          <string>Data Connections</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_Data_Connections">
          <item>
           <widget class="QScrollArea" name="scrollArea_Data_Connections">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents_Data_Connections">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>179</width>
               <height>157</height>
              </rect>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_Data_Connections_Buttons">
            <item>
             <widget class="QPushButton" name="pushButton_Data_Connections_Cancel">
              <property name="text">
               <string>Cancel</string>
              </property>
              <property name="default">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_Data_Connections_Add">
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignTop">
             <widget class="QPushButton" name="pushButton_Data_Connections_Save">
              <property name="text">
               <string>Save</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_Plugin_Config">
         <attribute name="title">
          <string>Plugin Config</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_Plugin_Config">
          <item>
           <widget class="QScrollArea" name="scrollArea_Plugin_Config">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents_Plugin_Config">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>174</width>
               <height>157</height>
              </rect>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_Plugin_Config_Buttons">
            <item>
             <widget class="QPushButton" name="pushButton_Plugin_Config_Cancel">
              <property name="text">
               <string>Cancel</string>
              </property>
              <property name="default">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_Plugin_Config_Upload">
              <property name="text">
               <string>Upload</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_Plugin_Config_Store">
              <property name="text">
               <string>Store</string>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignTop">
             <widget class="QPushButton" name="pushButton_Plugin_Config_Save">
              <property name="text">
               <string>Save</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBarBottom">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
border-top-color: rgb(103, 105, 105);
border-right-color: rgb(0, 0, 0);
border-bottom-color: rgb(0, 0, 0);
border-left-color: rgb(0, 0, 0);
color: rgb(250, 255, 255);</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBarTop">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
border-bottom-color: rgb(103, 105, 105);
border-right-color: rgb(0, 0, 0);
border-top-color: rgb(0, 0, 0);
border-left-color: rgb(0, 0, 0);
color: rgb(250, 255, 255);</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
